setup.go

```diff
@@ -151,9 +152,41 @@ func setup(c *caddy.Controller) error {
 				return errors.New("Parse error: dial_timeout cannot be negative.")
 			}
 			fp.dialTimeout = time.Second * time.Duration(timeout)
+		case "outgoing":
```

Comment:
Benefit of having a single policy is that we don't have to go a level deeper into this directive.